uiowa/uiowa
# Move stage_file_proxy origin setting to postBis instead of active config.

Follow up from #4673 

Using active config leaves sites in a config diff. Config ignoring just the stage_file_proxy.settings:origin did not work. This PR moves the setting to local settings file override after bis is run.

# How to test

- `blt bis`
- See docroot/sites/175.uiowa.edu/settings/local.settings.php has origin set to prod URL.
- See docroot/sites/sandbox.uiowa.edu/settings/local.settings.php has origin set to internal prod URL. That is set from the doroot/sites/sandbox.uiowa.edu/blt.yml file.
- Sync a few sites via `blt dsa`.
- `blt ume cst`
- There should be no config diffs.
